.github/workflows/cypress-tests-runner.yml

```diff
@@ -421,31 +404,39 @@ jobs:
       - name: Start mock servers for alpha connectors
         if: ${{ env.RUN_TESTS == 'true' }}
         run: |
-          # Start mock servers for each alpha connector
+          # Start mock servers for each alpha connector with unique ports
+          PORT_COUNTER=3010
           for connector in ${{ env.ALPHA_PAYMENTS_CONNECTORS }}; do
             pascal_case_connector="$(echo "$connector" | sed -E 's/(^|_)([a-z])/\U\2/g')"
             if [ -f "cypress-tests/cypress/e2e/configs/mock-server/${pascal_case_connector}.js" ]; then
-              echo "Starting mock server for ${connector}"
+              echo "Starting mock server for ${connector} on port ${PORT_COUNTER}"
```

Comment:
Why not start all mock servers in a single process, but have them accessible with a suitable prefix?

Say `localhost:3000/celero/...` and `localhost:3000/silverflow/...`?

Reply:
this would be significant refactoring, but it would be worth it to make the code more readable and maintainable.
the existing mock server code would have to be updated such that single server file starts the server and routes requests and responses to the appropriate handlers by exporting the handlers from the individual files.

Reply:
i would want to take this change up as a separate pr just so that the existing alpha integrations are unblocked (celero is waiting in line followed by multiple other alpha connectors).

thoughts?